src/Employee.java

```diff
@@ -0,0 +1,47 @@
+public class Employee {
+    private final String fullName;
+    private int department;
+    private float salary;
+    private final int id;
+    static int counter = 0;
```

Comment:
```suggestion
    private static int counter = 0;
```
счётчик должен быть внутренним и не должен показыавться наружу